exception `OutOfResourceException`
============================
__Package :__  `optima_v2.runtime`  
__Extends :__ [`Exception`](exception.md)

`OutOfResourceException` represents an exception thrown when the runtime failed to allocate resource.

This exception is thrown by the runtime in C++. On Python, users cannot create nor raise this exception.
